libbeat/processors/add_cloud_metadata/provider_aws_ec2.go

```diff
@@ -75,20 +74,16 @@ func fetchRawProviderMetadata(
 	client http.Client,
 	result *result,
 ) {
-	logger := logp.NewLogger("add_cloud_metadata")
-
 	// LoadDefaultConfig loads the EC2 role credentials
 	awsConfig, err := awscfg.LoadDefaultConfig(context.TODO(), awscfg.WithHTTPClient(&client))
 	if err != nil {
-		logger.Warnf("error loading AWS default configuration: %s.", err)
```

Comment:
Is not safer to change this to debug? (rather deleting sth that might be needed in troubleshooting)

Reply:
Since all the other providers don't have loggers, I thought we could keep the logic. And allowing debugging would be causing the same problem all over again

Reply:
> Since all the other providers don't have loggers, I thought we could keep the logic. And allowing debugging would be causing the same problem all over again

It is not really the same - all other providers are using the default [http fetcher](https://github.com/elastic/beats/blob/main/libbeat/processors/add_cloud_metadata/http_fetcher.go), but aws - [generic fetcher](https://github.com/elastic/beats/blob/main/libbeat/processors/add_cloud_metadata/generic_fetcher.go)